### DISCUSSION TITLE
Display new comments on a post

### DISCUSSION THREAD

**Discussion by StefanTraistaru** [Docile-Alligator/Infinity-For-Reddit#926] [Ideas]
I used to use Boost for reddit and there was an interesting feature that I really liked in that app.

When entering the comment section of a previously viewed post the app displays a toast in which it displays the number of new comments that have been made in that post since I last viewed it. Also, when scrolling, the new comments are highlighted. This also works when simply pulling down to refresh comments.

Usecase explained:
I sometimes read the comments on a post in rising/new and then save it for later. When I enter the post again, I can see how many new comments there are (if any) and easily see them because they are highlighted. This way I can easily get back into a post and see what new answers or discussions have appeared.

For example:
1. I view a post with 2 comments.
2. Save it for later (so I can find it) and go back to my feed.
3. Spend some time in the app. I can even close the app.
4. Go back to that post and a toast shows up saying: 5 new comments
5. When scrolling through the comments, these 5 new comments are highlighted and I can find them easily.
6. Without exiting the post, I pull down to refresh the comment section and a toast shows up: 1 new comment (or maybe none, it depends)
7. When scrolling through the comments, those 5 new comments from step 4 and 5 are not highlighted anymore. Only this new one that was found at step 6.

Screenshot from Boost for reddit as an example:
The new comments are the ones with the grey background (highlighted). If I were to refresh again, they will apear with a black brackground just like the others.
![Screenshot_20210108-160705_resized](https://user-images.githubusercontent.com/23136219/104033103-c2072a00-51d7-11eb-85df-a8eeaafe6f37.jpg)

**Comment by shreya-shah** [comment]
Yes. This would be a very helpful feature. Please do consider it
